chapters/operatorsandexpressions.tex

```diff
@@ -433,6 +433,13 @@ \subsection{Numeric Functions and Conversion Functions}\label{numeric-functions-
 Expands into \lstinline!noEvent(if $v$ >= 0 then $v$ else -$v$)!.
 Argument $v$ needs to be an \lstinline!Integer! or \lstinline!Real! expression.
 \end{semantics}
+\begin{nonnormative}
+The reason this does not generate events is that in many cases that would introduce more problems than the event would solve.
+A typical case is a flow equation of the form \lstinline!abs(x)*x=y!.
+If we have to solve for \lstinline!x! and had events for \lstinline!abs(x)! we would not find a solution if \lstinline!y! passes through zero.
```

Comment:
Maybe this claim is a bit too strong in order to be supported by a brief explanation?  Isn't it easier to simply state the example, perhaps with a statement about desirable property `abs(x) >= 0` for all `x`?

To me, the example also appears a little bit difficult to understand without some extra help.

```suggestion
By not generating events the property \lstinline!abs($x$)! $\geq 0$ for all $x$ is ensured at the cost of having a derivative that changes discontinuously between events.

A typical case requiring the event-free semantics is a flow equation of the form \lstinline!abs(x) * x = y!.
With event generation, the equation switches between the two forms \lstinline!x^2 = y! and \lstinline!-x^2 = y! at the events, where the events will not be coinciding exactly with the sign changes of \lstinline!y!.
When \lstinline!y! passes through zero, neither form of the equation will have a solution in an open neighborhood of \lstinline!y! $= 0$, and hence solving the equation has to fail at some point sufficiently close to \lstinline!y! $= 0$.
Without event generation, on the other hand, the equation can be solved easily for \lstinline!x!, also as \lstlinline!y! passes through zero.
```